06_gpu_and_ml/audio-to-text/parakeet.py

```diff
@@ -0,0 +1,282 @@
+# ---
+# cmd: ["modal", "serve", "06_gpu_and_ml/audio-to-text/parakeet.py::server_app"]
```

Comment:
If you collapse this into one app, as described below, this ends up becoming just `modal run`, which is the default `cmd`, and so you can elide this field -- and the entire frontmatter!